### DISCUSSION TITLE
How to move line item Terminal to top and always open this at launch of Geany?

### DISCUSSION THREAD

**Discussion by penalvch** [geany/geany#3705] [Q&A]
Hi all,

Love Geany so far. However, I wanted to inquire on how to move the Terminal to the top and always open this at launch of Geany? Basically, the screenshot below. Any help is appreciated. Thanks!
![image](https://github.com/geany/geany/assets/115644052/30104a44-6d9d-45c0-9f15-1fcdf754cc85)

**Comment by elextr** [comment]
There is no existing way to move the terminal to the top, and I'm not sure what that will gain, but it may break code that assumes status is first.  Also note the terminal is not available on Windows and is optional on Linux.

If "somebody" made a well written PR to save and restore the current message window tab it would likely be accepted.
